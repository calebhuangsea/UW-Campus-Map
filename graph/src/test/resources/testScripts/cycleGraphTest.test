# Test add 2 edges for 2 nodes(cycle)
# Tests that a graph can be created
# 2 nodes can be added to the graph
# 2 cycle edges can be added to 2 node

# Create a graph
CreateGraph graph1

# Add one nodes
AddNode graph1 n1
AddNode graph1 n2

# Add an edge for one edge
AddEdge graph1 n1 n2 e1
AddEdge graph1 n2 n1 e2

# Print the node in the graph and the outgoing edges from n1
ListNodes graph1
ListChildren graph1 n1
ListChildren graph1 n2