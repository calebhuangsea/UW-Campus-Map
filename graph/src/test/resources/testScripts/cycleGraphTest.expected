# Test add 2 edges for 2 nodes(cycle)
# Tests that a graph can be created
# 2 nodes can be added to the graph
# 2 cycle edges can be added to 2 node

# Create a graph
created graph graph1

# Add one nodes
added node n1 to graph1
added node n2 to graph1

# Add an edge for one edge
added edge e1 from n1 to n2 in graph1
added edge e2 from n2 to n1 in graph1

# Print the node in the graph and the outgoing edges from n1
graph1 contains: n1 n2
the children of n1 in graph1 are: n2(e1)
the children of n2 in graph1 are: n1(e2)